index.html

```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
           </p>
         </div>
         <div class="project-tag">
+          <ul class="project-skills flex">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+        <button class="see-project-btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about" class="about">
+    <div class="about-container flex column">
+      <div class="abt-me flex column">
+        <h2 class="header-text">About <br>Myself</h2>
+        <p class="para-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="self-container">
+          <p>Let's Connect</p>
+          <div class="list-nav">
+            <ul class="list-items skill-items">
+              <li>
+                <a href="">
+                  <img src="images/Twitter.svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img src="images/Vector.svg" alt="my social handle images"/><img src="images/Vector (1).svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
```

Comment:
Your design contains these background images. Figma design does not contain such images.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219857220-f375b1e8-8b25-4162-a452-983d463790b9.png)| ![image](https://user-images.githubusercontent.com/61361037/219857242-62e483e6-a55a-443c-9e94-b1c4a2416cb0.png)|